src/network/network_manager.cpp

```diff
@@ -22,21 +22,21 @@
 #include "common/thread_pool.h"
 
 namespace peloton {
-namespace wire {
+namespace network {
 
-int LibeventServer::recent_connfd = -1;
```

Comment:
recent_connfd is a static member in LibeventServer. And this line is actually variable initialization.
The reason they are doing this is that c++ forbids static member initialization within the class.